src/generic.js

```diff
@@ -0,0 +1,48 @@
+import styled from "tachyons-components";
```

Comment:
is this a file from the library? maybe a better file name to make clear what this does?

Reply:
yes. I just rename the file `loadingStyle`